flight_phase_files/HALO_RF04_20200126_info.yaml

```
@@ -21,71 +26,366 @@ remarks:
 - Four coordinated straight legs over Meteor.
 - Cirrus clouds in 14-17 km present for almost full flight.
 segments:
-- kind: straight_leg
+- kinds:
+  - straight_leg
```

This segment should also be kind "super_curtain"
```suggestion
  - straight_leg
  - super_curtain
```